experimenter/experimenter/nimbus-ui/src/components/Summary/TableAudience/index.tsx

```diff
@@ -142,6 +141,21 @@ const TableAudience = ({ experiment }: TableAudienceProps) => {
                 {experiment.isFirstRun ? "True" : "False"}
               </td>
             </tr>
+            <tr>
+              {experiment.targetingConfigSlug && (
+                <>
+                  <th>Advanced Targeting</th>
+                  <td data-testid="experiment-target">
+                    {displayConfigLabelOrNotSet(
+                      experiment.targetingConfigSlug,
+                      targetingConfigs,
+                    )}
+                  </td>
+                </>
+              )}
+              <th></th>
+              <td></td>
+            </tr>
             {experiment.jexlTargetingExpression &&
```

Comment:
you don't need to do that you can do col span

Reply:
That was existing code just moved down. I think we should re-write this whole file (since it's using things like that elsewhere too) and I left it in since it was out of scope for this work

Reply:
no I mean the empty tag  you added`<td></td>` and <th></th> as they are empty and you want to use one row, so consume all columns, like we use it, if you will see the next row it uses colspan    
```javascript 
{experiment.jexlTargetingExpression &&
            experiment.jexlTargetingExpression !== "" ? (
              <tr>
                <th>Full targeting expression</th>
                <td
                  colSpan={3}
```
I don't think we do elsewhere, I re-wrote the whole file when I was designing :D, I hope it shouldn't be doing that, but let me know If it does somewhere else too :D

Reply:
oh I see I see !